smoke.sh

```diff
@@ -68,6 +72,8 @@ function finish ()
     if [ $RET -ne 0 ]; then
         cat /build/dbench-logs || true
     fi
+    #Move statedumps to be archived
+    mv /var/run/gluster/*dump* /var/log/glusterfs/
```

Comment:
What I mean is to do this at the start of the test:
```
gluster volume set test-volume server.statedump-path /var/log/glusterfs
```
This avoids having to perform an mv at all.